### PR TITLE
If there are more instrumentation enhancers included, the last one will disable others

### PR DESCRIPTION
**Pull request by zalmoxisus**
Related to https://github.com/reactjs/redux/pull/1657.

### CODE REVIEW

File: src/instrument.js [hunk 1]
```diff
@@ -420,7 +420,8 @@ export default function instrument(monitorReducer = () => null, options = {}) {
     );
   }
 
-  return createStore => (reducer, initialState, enhancer) => {
+  return createStore => (reducer, initialState, enhancer, disabled) => {
```
Comment (gaearon) [inline src/instrument.js]: Hmm. I don’t feel good about enforcing a very specific meaning that only makes sense for this particular use case to a generic `createStore` signature.

**Comment by gaearon** [comment]
Maybe we can set some flag on the returned store object instead, or on the lifted reducer.

**Comment by zalmoxisus** [comment]
@gaearon, it would work easily if we'd need the first enhancer to disable the next ones, but not here.
So in this scenario we should whether let `liftReducer` and `unliftStore` to work twice (for both enhancers) or to generate UUID for every enhancer and allow those functions only for the current UUID. Both solutions seem too hacky, unless I'm not missing something.

**Comment by gaearon** [comment]
Let’s close this for now and revisit after #1702.

**Pull request closed without merging**